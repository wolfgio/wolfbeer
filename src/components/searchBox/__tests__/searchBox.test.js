import React from 'react';
import { create, act } from 'react-test-renderer';

import SearchBox from '../searchBox';

describe('<SearchBox />', () => {
  it('should render without error', () => {
    let root;
    act(() => {
      root = create(<SearchBox />);
    });

    expect(root.toJSON()).toMatchSnapshot();
  });
});
