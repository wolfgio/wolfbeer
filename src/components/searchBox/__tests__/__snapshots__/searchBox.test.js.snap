// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBox /> should render without error 1`] = `
<div
  className="sc-AykKC gIAcmC"
>
  <img
    className="sc-AykKD hZucOd"
    src={
      Object {
        "process": [Function],
      }
    }
  />
  <input
    className="sc-AykKE jbwMtV"
    id="autocomplete"
    onFocus={[Function]}
    placeholder="Pesquisar endereÃ§o..."
    type="search"
  />
</div>
`;
